client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -343,6 +343,57 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
       handleReportShuffleFetchFailure(context, appShuffleId, shuffleId)
   }
 
+  def setupEndpoints(
+      slots: WorkerResource,
+      shuffleId: Int,
+      connectFailedWorkers: ShuffleFailedWorkers): Unit = {
+    val futures = new util.LinkedList[(Future[RpcEndpointRef], WorkerInfo)]()
+    slots.asScala foreach { case (workerInfo, _) =>
```

Comment:
```suggestion
    slots.asScala.foreach { case (workerInfo, _) =>
```

https://github.com/databricks/scala-style-guide?tab=readme-ov-file#infix-methods